Bio/UniProt/__init__.py

```diff
@@ -11,3 +11,33 @@
 See also Bio.SwissProt and the "swiss" support in Bio.SeqIO for
 the legacy plain text sequence format still used in UniProt.
 """
+
+from Bio._py3k import urlopen as _urlopen
+from Bio._py3k import urlencode as _urlencode
+from Bio._py3k import _binary_to_string_handle
+
+
+def search(text, output_format="tab", sort="score", oragnism="", columns=(),
+           isoform=False, compress=False, offset=0, limit=0):
+    """Perform a query over the UniProt API.
+
+    More at: https://www.uniprot.org/help/api_queries
```

Comment:
We should probably provide a little more help here, specifically that columns should be a list/tuple of strings, and perhaps that isoform and compress are treated as booleans, with offset and limit expected to be integers.